### PR TITLE
[CORE-API] Add Meal Entity, controller and service

### PR DESCRIPTION
**Pull request by aacgn**
Solves #12

### CODE REVIEW

**Review by eduardosm7 — APPROVED** [review]

**Review by viniciusgiles — CHANGES_REQUESTED** [review]
There should be a 1-N relation between the Place end Meal entities. Please refer to https://github.com/typeorm/typeorm/blob/master/docs/many-to-one-one-to-many-relations.md if you need any help.

**Review by mmp-s — APPROVED** [review]

**Comment by aacgn** [comment]
1-N relation between the Place end Meal entities made.

File: core-api/src/services/meal.service.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+     * @param place_id
+     */
+    public static async create(name: string, description: string, price: number, place_id: number): Promise<SafeMeal> {
+        const repository = db.getRepository(Meal);
+
+        const place = await PlaceService.findByIdWithoutSafety(place_id);
+
+        const newMeal = new Meal(name, description, price, place);
+
+        await this.validateFields(newMeal);
```
Comment (eduardosm7) [inline core-api/src/services/meal.service.ts:27]: This call is not validating whether the place exists or not. You can check this method in place.service, it does a similar thing.

Comment (aacgn) [reply core-api/src/services/meal.service.ts:27]: Done

**Review by viniciusgiles — APPROVED** [review]

**Review by eduardosm7 — APPROVED** [review]

**Pull request merged**